views/widgets/mainNav.ejs

```diff
@@ -40,6 +40,7 @@
                 </div>
             </li>
             <li class="nav-item"><a href="/smallgroups" class="nav-link">Small Groups</a></li>
+            <li class="nav-item"><a href="/ministryteams" class="nav-link">Ministry Teams</a></li>
```

Comment:
I was thinking maybe making another dropdown menu called "Get Involved" or "Serve" or something, and including Small Groups and Ministry teams in it. if it doesn't take too much time, could you do that? Otherwise I can assign it as a small task to another person after this gets merged.